draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -197,6 +198,8 @@ DNSSEC {{I-D.ietf-dnsop-dnssec-bcp}} SHOULD be employed by the domain owner to p
 
 DNSSEC validation MUST be enabled by service providers that verify domain verification records they have issued and when no DNSSEC support is detected for the domain owner zone, SHOULD attempt to query and confirm by matching the validation record using multiple DNS validators on (preferably) unpredictable geographically diverse IP addressses to reduce an attackers capability of DNS spoofing. Alternatively, service providers MAY perform multiple queries spread out over a longer time period to reduce the chance of receiving spoofed DNS answers.
 
+Providers MUST NOT verify domains above the Public Suffix {{Section 9 of RFC8499}}.
```

Comment:
I don't understand this sentence.
For example, at $dayjob we have put aivencloud.com into the PSL, because we assign entries in it to customers.
But it is not really "public" and I don't know how to interpret "above" in this case?

Reply:
I think the point here was that Google should not be delegating control of something like .com. But yeah I know the text could be clearer. Also perhaps it's obvious?

Reply:
Was there any resolution to this PR? Erik Nygren has also made a comment that we should exclude validation of public suffixes, which seems reasonable to me.

Paul - if aivencloud.com is in the PSL, then I think we can argue that its a public suffix, even if it's just for your own customers. 

Since your customers can register names under aivencloud.com, I think what we want to ensure is that: one of your (potential) customers cannot deploy a domain validation record for aivencloud.com itself, e.g. by registering <challenge_record>.aivencloud.com/TXT, or aivencloud.com/TXT. That seems like a reasonable precaution we should take.